### PR TITLE
Reclone step repo. Continue comments feature.

### PR DESCRIPTION
**Pull request by saraholiviawu**
Note that the history of commits from Week 3 and earlier can be found in a separate repository called 'step.'

### CODE REVIEW

**Review by saraholiviawu — COMMENTED** [review]
This commit includes a delete feature in the comments section. Still need to add functionality to delete multiple comments at once.

**Review by saraholiviawu — COMMENTED** [review]
Screenshot 2020-06-06 at 10 10 30 PM - Display 2

**Review by saraholiviawu — COMMENTED** [review]
Video snippet of multiple comment deletions. Note that you can delete several at once despite having entered a comment following the deletion(s).
https://drive.google.com/file/d/1VcZC9xdO6qtiwNnV5gN848B3wp2SRdxf/view

**Review by saraholiviawu — COMMENTED** [review]
See video snippet for demonstration of 5-comment display limit.
https://drive.google.com/file/d/1ecRnwM1n7nH6T3bgdClFzGel3XfJtJ9y/view

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -26,7 +38,83 @@
...
+      comments.add(singleComment);
+    }
+
+    // Ensure maxComments is not greater than the existing number of comments
+    if (comments.size() < reqComments) {
+     reqComments = comments.size();
+    }
+    
+    // Remove comments that do not fit within maxComments
+    for (int i = comments.size()-1; i > reqComments-1; i--) {
```
Comment (peter14f) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: add spacing before and after the subtract operator
```
for (int i = comments.size() - 1; i > reqComments - 1; i--)
```

Comment (saraholiviawu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fix in following commit

**Review by sabadas — APPROVED** [review]

**Pull request merged**